Directory.Build.props

```diff
@@ -23,7 +23,7 @@
   <ItemGroup>
     <!-- All projects should use Nerdbank.GitVersioning for consistent version numbers -->
     <PackageReference Include="Nerdbank.GitVersioning">
-      <Version>3.3.37</Version>
```

Comment:
Please rebase on the latest `main` branch; I've since merged in your PR (#584) to update this package.

Reply:
Done.